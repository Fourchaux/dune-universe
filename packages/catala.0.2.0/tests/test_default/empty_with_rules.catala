@Article@

/*
new scope A:
  param x content int
  param y content int

scope A:
  def y := 1
  def x [y = 2] := 1
  def x [y = 3] := 1
  def x [y = 4] := 1
*/