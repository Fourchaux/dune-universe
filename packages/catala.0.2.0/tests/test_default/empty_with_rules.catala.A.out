[ERROR] This variable evaluated to an empty term (no rule that defined it applied in this situation)
[ERROR] 
[ERROR]  --> test_default/empty_with_rules.catala
[ERROR]   | 
[ERROR] 5 |   param x content int
[ERROR]   |         ^
